advisories/a88042.md

```diff
@@ -0,0 +1,53 @@
+
+This issue primarily affects incremental backups taken after an `IMPORT INTO` operation is reverted. For this issue to occur, the follow series of conditions need to be met:
+
+1. First, a table must be backed up, typically as part of a full-cluster backup, while it is offline and being written to by an ongoing `IMPORT INTO` job. Rows which are being written to the table by the in-progress `IMPORT INTO` job are included in this backup.
+1. Next, that `IMPORT INTO` job must fail or be canceled. This failure or cancellation triggers the roll-back process, which reverts rows which it had written, and then brings the table back online.
+1. After the `IMPORT INTO` rollback completes and the table is back online, an incremental backup must be taken of the table. This incremental backup is appended to the original backup that was taken while the table was offline.
+1. When this incremental backup is restored using an affected CockroachDB version, rows which were initially written by the `IMPORT` and backed up, but then later reverted when the `IMPORT INTO` was canceled before the incremental backup, will incorrectly appear in the restored table.
+
+If a backup chain starts from a full backup subsequent to the `IMPORT INTO` job cancellation, it is unaffected.
+
+The technical details can be found in the following GitHub issues:
+
+- [#88042](https://github.com/cockroachdb/cockroach/issues/88042)
+- [#88043](https://github.com/cockroachdb/cockroach/issues/88043)
+
+
+## Statement
+
+This is resolved in CockroachDB by [#87312](https://github.com/cockroachdb/cockroach/pull/87312).
+
+The fix has been applied to the maintenance release of CockroachDB [v22.1.9](../releases/v22.1.html#v22-1-9).
+
+This public issue is tracked by [#87305](https://github.com/cockroachdb/cockroach/issues/87305).
+
+## Mitigation
+
+Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8) are encouraged to upgrade to [v22.1.9](../releases/v22.1.html#v22-1-9) or a later version.
+
+If a `RESTORE` must be run using prior versions, Cockroach Labs recommends that where possible, you restore only to full backups. If you must restore to an incremental backup taken on an affected version, restore only tables on which `IMPORT INTO` did not run or succeeded.
+
+The best mitigation, however, is to upgrade to v22.1.9 as soon as possible.
+
+## Impact
+
+`RESTORE` of incremental backups, which were created during a specific series of events relating to `IMPORT INTO` and cancellation, may include rows that should not be restored. This has been fixed in CockroachDB patch version v22.1.9, available in [Releases](https://www.cockroachlabs.com/docs/releases/). Versions affected include {{ page.affected_versions }}.
```

Comment:
Same nit as first comment: recommend including article and a slight rephrase.

Reply:
Also, TIL `{{ page.affected_versions }}` -- nice!

Reply:
That relies on a variable that's set in the page's metadata.